Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/WindowsMixedRealityUtilities.cs

```diff
@@ -88,14 +88,15 @@ public static HolographicFrame CurrentHolographicFrame
         {
             get
             {
+                if (WmrUtilitiesProvider == null)
+                {
+                    return null;
+                }
+
 #if DOTNETWINRT_PRESENT
```

Comment:
no ENABLE_DOTNET here?

Reply:
It may be needed here, but the existing .NET workaround was specific to marshaling the `ISpatialCoordinateSystemPtr`. It would need updating to support `IHolographicFramePtr`, and the DotNetWinRT adapter may provide a better workaround for .NET Native. Testing needed.